apps/web/src/pages/Auth/SignUpPage/SignUpPage.tsx

```diff
@@ -54,147 +120,680 @@ const SignUpPage = () => {
     <>
       <SEO title={pageTitle} />
       <Hero title={pageTitle} crumbs={crumbs} />
-      <div data-h2-padding="base(x3, 0)">
-        <div data-h2-container="base(center, small, x1) p-tablet(center, small, x2)">
-          {!iapMode ? (
-            <>
-              {/* Standard copy */}
-              <p>
+      <div
+        data-h2-container="base(center, large, x1) p-tablet(center, large, x2)"
+        data-h2-margin-bottom="base(x1)"
+      >
+        {!iapMode ? (
+          <>
+            {/* Standard copy */}
+            <Heading
+              Icon={MapIcon}
```

Comment:
I feel like this was fixed at some point. Not sure if it makes sense here but the icon in our `Heading` component should have `flex-shrink: 0'` or else it looks odd on mobile.

![Screenshot 2023-09-08 151156](https://github.com/GCTC-NTGC/gc-digital-talent/assets/4127998/cecc95a0-3b33-4a01-8052-59db4bd1bbb6)

Reply:
Good catch!  Contributed to the other branch.
[styling tweaks from dev review](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7835/commits/8fabb396dca6bcbf0dc5e32332e2e73c3034efda)